### PR TITLE
Extend tests for OpenBabelParser

### PR DESCRIPTION
**Pull request by lunamorrow**
This PR extends on #12 by adding new tests for different file/molecule types, in response to Issue #15

Currently, chainIDs are NOT supported -> please refer to Issue #17

I seem to have uncovered a bug with OpenBabel parsing of bonds defined in SDF files, but this requires more extensive testing to be sure. For now, the expected value of bonds for the SDF originating Universe has been set to 0 to pass tests, as the OBMol created from the SDF has 0 bonds recorded.

PR Checklist
------------
 - [X] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [X] Issue raised/referenced?

### CODE REVIEW

**Comment by lunamorrow** [comment]
Just realising I should have checked out a new branch for this PR... please disregard all of the old commits!

**Review by hmacdope — APPROVED** [review]
LGTM!

**Review by xhgchen — APPROVED** [review]

**Comment by xhgchen** [comment]
LGTM!

**Comment by lunamorrow** [comment]
@pep8speaks

File: mda_openbabel_converter/tests/test_openbabel_parser.py [hunk 1]
```diff
@@ -113,3 +109,119 @@ def test_mass_check(self, top, filename):
...
+    expected_n_residues = 1
+    expected_n_segments = 1
+    expected_n_bonds = 50
+
+    def test_attrs_total_counts(self, top):
+        u = mda.Universe(top, format = 'OPENBABEL')
+        ag = u.select_atoms("all")
+        res = ag.residues
+        seg = ag.segments
+        print(len(ag))
```
Comment (cbouy) [inline mda_openbabel_converter/tests/test_openbabel_parser.py:224]: image

**Pull request merged**